src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -687,6 +700,38 @@ namespace
 
         return true;
     }
+
+    bool write_extra_aovs(
+        const ImageStack & images,
```

Comment:
Remove space between the types and the '&' and align parameter names vertically.